src/status_im/contexts/communities/actions/request_to_join/view.cljs

```diff
@@ -23,10 +23,8 @@
 (defn- view-internal
   [{:keys [theme]}]
   (fn []
-    (let [{:keys [name
-                  id
-                  images]} (rf/sub [:get-screen-params])
-          {:keys [color]}  (rf/sub [:communities/community id])]
+    (let [{:keys [id]}                (rf/sub [:get-screen-params])
+          {:keys [color name images]} (rf/sub [:communities/community id])]
```

Comment:
FYI: It's buggy to rely on `:get-screen-params` for dynamic data because we 99% of the close over values within an event handler in the view, hence if `name` or `images` change in the community instance in the app-db while the user is on this screen, the UI won't update.